src/core/server/metrics/metrics_service.ts

```diff
@@ -37,7 +37,7 @@ export class MetricsService
   private readonly logger: Logger;
   private metricsCollector?: OpsMetricsCollector;
   private collectInterval?: NodeJS.Timeout;
-  private metrics$ = new ReplaySubject<OpsMetrics>(1);
+  private metrics$ = new Subject<OpsMetrics>();
```

Comment:
With the metrics being reset after every emission, we probably don't want a replay effect anymore, so I switched back to a plain Subject. This also go closer to the legacy/oppsy implementation that was using events (therefor have no replay capabilities)

Reply:
It means that a subscriber doesn't receive the last emitted value. But okay, if it works in this way in LP